### ISSUE TITLE
Workitem Links Enhancements Suggestions

### ISSUE THREAD

**Issue by sheldonhull** [ostreifel/Links-Group#11]
**Describe the bug**
This resource is great! It really solved an issue I had identified for those using kanban but wanting to easily create a checklist of progress inside that. So much easier to break down a workitem from right in the workitem form.

**Expected behavior**
Would like to suggest a usability set of improvements that would really help this be even more useful for team members. 

1. Setting for filtering to only childlinks. Since the use case I have for this is to break down something into smaller tasks or items, seeing parent items listed above doesn't make sense, especially since those are immediately visible in the Built in Links section on the workitem form already. 
2. Allow setting 1-2 columns property in that checklist. For example: Assigned To, Completed Work, Remaining work all require me to go into the task each time, instead of easily summarizing from the link. Would be great to have those key bits available so I can immediately update and then mark done without more navigation.
3. If you want to show more than the childlinks, perhaps subgroup by "Child Items", "Parent Items" or something similar to make it understandable. It confused me as a user the first time when I saw a product backlog item links showing a feature next to tasks.

**Additional context**
Fantastic extension. Of all the one's I've setup for my team, this is probably my favorite!

**Comment by kristofferlindvall** [comment]
I agree. I only want to see children. Not related or other relationships.
